osmaxx/api_client/API_client.py

```diff
@@ -66,31 +56,6 @@ def authorized_post(self, url, json_data=None, **kwargs):
         self._make_request_authorized()
         return self.post(url, json_data, **kwargs)
 
-    def authorized_options(self, url, **kwargs):
-        self._make_request_authorized()
-        return self.options(self._to_fully_qualified_url(url), **kwargs)
-
-    def service_is_available(self):
-        try:
-            requests.get(self.service_base)
-            is_available = True
-        except ConnectionError as e:
-            logger.error('service {service_url} is down'.format(service_url=self.service_base), e)
-            is_available = False
-        return is_available
-
-    def _get_error(self, response):
-        try:
-            response.raise_for_status()
-            error = None
-        except requests.HTTPError as e:
-            try:
-                error = e.response.json()
-            except:
-                error = {'unknown': 'Unknown error happened. Please try again.'}
```

Comment:
No, let's not do that again; this was only needed because we had to have an explicit response. Now it should just fail loudly IMO.